servers.json

```diff
@@ -1184,6 +1184,14 @@
             "scala.gg"
         ]
     },
+    {
+        "id": "certix",
+        "name": "Certix Network",
+        "addresses": [
+            "certix.cc",
+            "eu.certix.cc"
```

Comment:
Lunar Client services automatically detect sub-domains, please remove this.

Reply:
I have now removed the sub domain